src/league-of-legends-api/poll-current-game.ts

```diff
@@ -0,0 +1,56 @@
+import axios, { AxiosError, AxiosResponse } from 'axios';
+import { VoiceBasedChannel } from 'discord.js';
+import { playClip, setIntervalImmediately } from '../helpers';
+import fs from 'fs';
+import { AudioPlayer } from '@discordjs/voice';
+import { Player, RootGameObject } from './types/index';
+const https = require('https');
+
+const LOL_GAME_CLIENT_API = 'https://127.0.0.1:2999/liveclientdata';
+const TWO_SECONDS = 2000;
+
+const httpsAgent = new https.Agent({
+  rejectUnauthorized: false,
+  cert: fs.readFileSync('src/league-of-legends-api/riotgames.pem')
+});
+
+let cachedGame: RootGameObject;
+
+export const pollCurrentGame = async (
+  channel: VoiceBasedChannel,
+  audioPlayer: AudioPlayer
+) => {
+  setIntervalImmediately(async () => {
+    try {
+      const { data: latestGameResponse }: AxiosResponse<RootGameObject> =
+        await axios.get(`${LOL_GAME_CLIENT_API}/allgamedata`, { httpsAgent });
+
+      const previousKills: number[] =
```

Comment:
This seems to work well as an initial MVP event, but is pretty hard coded. I'm thinking we can use the Event array from the Riot API moving forward to build something more sophisticated. 

![image](https://user-images.githubusercontent.com/25968605/207192852-e1fce8f9-d129-4c13-a99d-4196b62aa75a.png)

Reply:
Yeah this is where we'd get the multi-kill data anyways so makes sense to use that. I think looking at events we'd only have to keep track of the number of events in the previous game data to determine if there are new ones and then parse those.